canonical-web-and-design/ubuntu.com
# Cleanup takeovers

## Done

- Remove old takeovers and take screenshots

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/0_docs/takeovers and check the snapshots


## Issue / Card

Fixes #8651 

## Screenshots
